meta/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+galaxy_info:
+  author: haslersn
+  description: Role for https://github.com/librespot-org/librespot
```

Comment:
Please the scribe what librespot is, instead of the URL

Reply:
It is described by the librespot project owners at that URL. The project owners are best at describing what the project does.